cl-postgres/public.lisp

```diff
@@ -79,12 +79,29 @@ currently connected."
                         :format :binary
                         :remote-filename path)))
 
+
+(defun get-host-address (host)
```

Comment:
Shouldn't this be conditional on #+sbcl to prevent warnings on other implementations?